integration/testdata/with_bp_composer_version/plan.toml

```diff
@@ -0,0 +1,7 @@
+[[requires]]
+name = "composer"
+
+  [requires.metadata]
+    launch = true
+    version = "9.9.9"
+    version-source = "BP_COMPOSER_VERSION"
```

Comment:
Nitpick: I think for this test it would be easier/clearer for the reader if you re-used the basic test fixture from the build_launch_test, and in the pack build in the test command you used the `.WithEnv()` function to add `BP_COMPOSER_VERSION: 9.9.9`.